optuna/integration/keras.py

```diff
@@ -57,6 +59,10 @@ def on_epoch_end(self, epoch: int, logs: Optional[Dict[str, float]] = None) -> N
         logs = logs or {}
         current_score = logs.get(self._monitor)
         if current_score is None:
+            message = "There is not an evaluation metric for pruning named {}.".format(
+                self._monitor
+            )
+            warnings.warn(message)
```

Comment:
Could you add a test case to check this warning to `tests/integration_tests/test_keras.py`?
We can use `with pytest.warns(UserWarning)` to assert the warning.